Externals/hidapi/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(hidapi)
+
+add_library(hidapi hidapi/hidapi.h)
```

Comment:
You're right! It breaks pretty much immediately on macOS. Not worth looking into right now.